### PR TITLE
feat: supports agent events

### PR DESCRIPTION
**Pull request by xzchaoo**
# Which issue does this PR close?

Closes #

# Rationale for this change
Agent now reports important events to HoloInsight server, which makes it better for users to known the running status of the log monitor items.

# What changes are included in this PR?
1. Add a GRPC method for registry
2. HoloInsight server now prints the events to log files

# Are there any user-facing changes?
No

# How does this change test
Local test

### CODE REVIEW

File: server/registry/registry-grpc-gen/src/main/proto/common.proto [hunk 1]
```diff
@@ -78,6 +78,8 @@ message CommonRequestHeader {
   string apikey = 1;
   string trace_id = 2;
   map<string, string> header = 3;
+  // workspace
+  string workspace = 4;
```
[the remark this answers is not in the document]
Comment (xzchaoo) [reply server/registry/registry-grpc-gen/src/main/proto/common.proto:82]: tenant is determined by apikey.

File: server/common/common/src/main/java/io/holoinsight/server/common/TrafficTracer.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  @Override
+  public void outboundUncompressedSize(long bytes) {
+    outboundUncompressedSize += bytes;
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * TCP 传输入流量
```
[the remark this answers is not in the document]
Comment (xzchaoo) [reply server/common/common/src/main/java/io/holoinsight/server/common/TrafficTracer.java]: fixed

File: server/common/common/src/main/java/io/holoinsight/server/common/TrafficTracer.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+   */
+  @Override
+  public void inboundWireSize(long bytes) {
+    this.inboundWireSize += bytes;
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * 解压后的入流量
```
[the remark this answers is not in the document]
Comment (xzchaoo) [reply server/common/common/src/main/java/io/holoinsight/server/common/TrafficTracer.java]: fixed

**Review by masaimu — APPROVED** [review]
LGTM

File: server/registry/registry-grpc-gen/src/main/proto/registry-for-agent.proto [hunk 4]
```diff
@@ -231,6 +231,35 @@ message MetaSync {
...
+    map<string, int64> numbers = 6;
+    // event strings
+    map<string, string> strings = 7;
+    // event logs
+    repeated string logs = 8;
+    string json = 9;
+  }
+}
+
 // 有一个问题, grpc的client能否感知到连接曾经断开? 一般来说当重新连上某个reg时, 需要重新执行 get_auth_info, register_agent
```
Comment (jiwliu) [inline server/registry/registry-grpc-gen/src/main/proto/registry-for-agent.proto:263]: Do these annotations need to be kept

**Pull request merged**